cmd/service/init.go

```diff
@@ -140,12 +141,13 @@ func getTemplateResult(result string, templates []*templateStruct) (tmpl *templa
 }
 
 func downloadTemplate(tmpl *templateStruct) (path string, err error) {
-	path, err = createTempFolder()
+	path, err = ioutil.TempDir("", "mesg-")
 	if err != nil {
 		return "", err
 	}
 
-	return path, gitClone(tmpl.URL, path, "Downloading template "+tmpl.Name+"...")
+	fmt.Printf("Downloading template %s ...\n", tmpl.Name)
```

Comment:
forgot to use _utils.ShowSpinnerForFunc()_? it was defined in previous version of _gitClone()_